desktop/renderer.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+const { app, BrowserWindow } = require("electron");
+const path = require("path");
+const { env } = require("process");
+
+// Keep a global reference of the mainWindowdow object, if you don't, the mainWindowdow will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow = null;
+let subpy = null;
+
+const PY_DIST_FOLDER = "dist-python"; // python distributable folder
+const PY_SRC_FOLDER = "../desktop"; // path to the python source
+// temporary hack to get the dist exe to run in the right place
```

Comment:
it is actually a persistent hack now that we don't use onefile mode for PyInstaller anymore